src/dataServices/dynamoDbDataService.ts

```diff
@@ -149,12 +149,7 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
     async updateResource(request: UpdateResourceRequest) {
         const { resource, resourceType, id } = request;
         const resourceCopy = { ...resource };
-        const getResponse = await this.readResource({ resourceType, id });
-        const currentVId: number = getResponse.resource.meta
-            ? parseInt(getResponse.resource.meta.versionId, 10) || 0
-            : 0;
-
-        resourceCopy.meta = generateMeta((currentVId + 1).toString());

```

Comment:
Where are we incrementing the version number now that we removed this?

Reply:
The `vid` is being updated here.
https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/blob/4f433d2eacdd81c25bdc6e5a2d5e9ea755a33204/src/dataServices/dynamoDbBundleServiceHelper.ts#L68